api-controllers/ServiceBrokerApiController.js

```diff
@@ -424,12 +436,20 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(NotFound, gone);
   }
 
-  static getDashboardUrl(plan, instanceId) {
+  getDashboardUrl(context) {
+    if (_.get(context, 'plan.manager.settings.dashboard_url_template') !== undefined) {
+      const urlTemplate = new Buffer(context.plan.manager.settings.dashboard_url_template, 'base64');
+      const dashboardUrl = encodeURI(_.template(urlTemplate)(context));
+      if (CONST.REGEX_PATTERN.URL.test(dashboardUrl)) {
+        return dashboardUrl;
+      }
+      throw new errors.UnprocessableEntity(`Unable to generate valid dashboard URL with the template '${urlTemplate}'`);
```

Comment:
Didnt understand this? `UnprocessableEntity` is a error class in the above stmt, which one do you want to put it in constant? Can you put that code snippet in here?

Reply:
I mean conventionally we define errors on top of class.
```js
const UnprocessableEntity = errors.UnprocessableEntity;
new UnprocessableEntity(....)
```

Reply:
Generally the usage is if there is more than one error you are using in your class then you have ref of errors and refer the other error classes via errors so that you dont have to import each error individually. Using each error also works but its verbose. Dont see how one is better than the other. 
Only scenario is if in the entire class i have only a specific type of error, then i generally use the way you have mentioned above. 
Can refer existing code for similar code.